cockatrice/src/gameselector.h

```diff
@@ -23,6 +23,7 @@ private slots:
     void actCreate();
     void actJoin();
     void checkResponse(const Response &response);
+    void updateButtonChoices(const QModelIndex &);
```

Comment:
Sorry, i don't get what you mean. I just said that right now there is no code convention defined about naming actions `actXXX`, but that as you suggested we should try to start using it and formalize it in the CONTRIBUTING.md file, that contains code best practices.